### PR TITLE
Pia 2522 camera

### PR DESCRIPTION
**Pull request by zladzeyka**
Allow camera customizations: 
 -  camera container view background color,
 -  camera preview frame color,
 -  camera buttons view background color.

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md [hunk 1]
```diff
@@ -70,12 +70,15 @@ Some background and text colors use the `GiniColor` type with which you can set
...
 ##### 2. Camera preview
+- Preview frame color &#8594;  `GiniConfiguration.cameraPreviewFrameColor`
 - Guides color &#8594;  `GiniConfiguration.cameraPreviewCornerGuidesColor`
 - Focus large image &#8594; <span style="color:#009EDF">*cameraFocusLarge*</span> image asset
 - Focus large small &#8594; <span style="color:#009EDF">*cameraFocusSmall*</span> image asset
 - Opaque view style (when tool tip is shown)  &#8594;  `GiniConfiguration.toolTipOpaqueBackgroundStyle`
 
 ##### 3. Camera buttons container
+- Background color &#8594;  `GiniConfiguration.cameraButtonsViewBackgroundColor`
+- Container view background color under the scroll indicator  &#8594;  `GiniConfiguration.cameraContainerViewBackgroundColor`
```
Comment (a-szotyori) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:81]: Where is there a scroll indicator on the camera screen? 🤔

Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:81]: IMG_6653
Bottom area with `scroll indicator`.

Comment (a-szotyori) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:81]: That one is called the `home indicator`: https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/

Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:81]: Thanks. I'll fix it.

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**